climpred/bootstrap.py

```diff
@@ -307,29 +329,58 @@ def bootstrap_compute(
                 reference,
                 metric=metric,
                 comparison=comparison,
+                dim=dim,
                 add_attrs=False,
+                **kwargs,
             )
         )
         # compute persistence skill
-        pers.append(compute_persistence(smp_hind, reference, metric=metric))
+        if metric not in PROBABILISTIC_METRICS:
```

Comment:
So what happens if it's a probabilistic metric?

Reply:
I don’t know how to calculate a persistence skill for a probabilistic metric. The persistence forecast has also only one member...